packages/terra-framework-docs/src/terra-dev-site/test/worklist-data-grid/gridMaskedCellData.json

```
@@ -240,137 +240,137 @@
+        { "content": "--", "isSelectable":false },
+        { "content": "0.73", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "0/85", "isSelectable":false }
       ]
     },
     {
       "id": "22",
       "cells": [
-        { "content": "Potassium mEq/L"},
-        { "content": "4.5" },
-        { "content": "--" },
-        { "content": "--" },
-        { "content": "4.5" },
-        { "content": "--" },
-        { "content": "4.9" },
-        { "content": "--" },
-        { "content": "--" },
-        { "content": "4.6" }
+        { "content": "Potassium mEq/L", "isSelectable":false },
+        { "content": "4.5", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "4.5", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "4.9", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "4.6", "isSelectable":false }
       ]
     },
     {
       "id": "23",
       "cells": [
-        { "content": "Troponin ng/mL"},
-        { "content": "0.03" },
-        { "content": "--" },
-        { "content": "--" },
-        { "content": "0.04" },
-        { "content": "--" },
-        { "content": "--" },
-        { "content": "0.04" },
-        { "content": "--" },
-        { "content": "0.03" }
+        { "content": "Troponin ng/mL", "isSelectable":false },
+        { "content": "0.03", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "0.04", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "0.04", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "0.03", "isSelectable":false }
       ]
     },
     {
       "id": "24",
       "cells": [
-        { "content": "Glucose mg/dL"},
-        { "content": "161" },
-        { "content": "--" },
-        { "content": "--" },
-        { "content": "162" },
-        { "content": "--" },
-        { "content": "179" },
-        { "content": "--" },
-        { "content": "--" },
-        { "content": "176" }
+        { "content": "Glucose mg/dL", "isSelectable":false },
+        { "content": "161", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "162", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "179", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "--", "isSelectable":false },
+        { "content": "176", "isSelectable":false }
       ]
     }
   ]
-}
+}
```

Nit: no newline error
```suggestion
}

```

---

Should we have gotten a linter error?